docs/examples/custom_operator.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime, timezone
+
+from flycs_sdk.custom_code import CustomCode, Dependency
+from flycs_sdk.entities import Entity
+from flycs_sdk.pipelines import Pipeline, PipelineKind
+from flycs_sdk.transformations import Transformation
+
+ENTITY_NAME = "my_dataset"
+
+transformations = []
+query = Transformation(
+    name="simple_copy",
+    query="SELECT * FROM raw.alpha.employees.employees AS raw",
+    version="1.0.0",
+    static=True,
+)
+transformations.append(query)
+
+
+def build(dag, env=None):
```

Comment:
@christophedcpm shouldn't we also include all the parameters in case it is inside of a parametrized pipeline?

Reply:
well, since this feature is only available using the python SDK and not using the yaml files, I don't think there is a real use of the parameters here.
ParametrizedPipeline are really just an easy way to generate multiple yaml pipelines files.